Documentation/PageTsconfig/Rte.rst

```diff
@@ -59,6 +72,30 @@ Consider the following Page TSconfig examples:
    # But disable RTE for tt_content bodytext again if the record type is "text"
    RTE.config.tt_content.bodytext.types.text.disabled = 1
 
+.. _pageTsRteOverridePreset:
+
+Example: Override preset
+------------------------
+
+Refer to the description of the order above for details of which setting has priority over which.
+
+Summary:
+
+*   Setting the preset via page TSconfig *for a specific field* overrides all
```

Comment:
I don't understand that sentence: "... overrides all else"

Reply:
If set, this is the setting which will be used, regardless of any other setting

Reply:
Added a comma, to make it clearer.